libnavigation-core/src/main/java/com/mapbox/navigation/core/MapboxNavigation.kt

```diff
@@ -239,7 +239,11 @@ constructor(
     }
 
     /**
-     * Return the current [TripSessionState].
+     * Return the current [TripSession]'s state.
+     * Could be either [STARTED] state when the session is active, running a foreground service and
+     * requesting and returning location updates or [STOPPED] state when the session is inactive.
```

Comment:
Yeah was thinking of a suggestion. Was going to say
```
     * State indicating when navigation is active or inactive.
     * @see [registerTripSessionStateObserver]
```

Also happy with the extra information provided here 💯.


possible rewording:

The state is [STARTED] when the session is active, running a foreground service and
requesting and returning location updates. The state is [STOPPED] when the session is inactive.